tests/test_pipeline.py

```diff
@@ -145,6 +148,67 @@ def test_dry_run():
         assert status == 0
 
 
+def test_python_paths():
+    # make a temp dir
+    with tempfile.TemporaryDirectory() as dirname:
+        os.mkdir(dirname + "/pretend")
+
+        # create a subdir of that with a module in
+        mod_dir = dirname + "/pretend"
+        mod_path = mod_dir + "/pretend_module.py"
+
+        # empty module, just to check it imports
+        open(mod_path, 'w').close()
+
+        assert os.path.exists(mod_path)
+        print(os.listdir(mod_dir))
+
+        # create a stage there that uses the submodule
+        stage_path = dirname + "/my_stage.py"
+        open(stage_path,  'w').write("""
+import ceci
+class MyStage(ceci.PipelineStage):
+    name = "MyStage"
+    inputs = []
+    outputs = []
+    config_options = {"x": int}
+    def run(self):
+        import pretend_module
+        assert self.config["x"] == 17
+""")
```

Comment:
what is this? if it's a comment should we remove it?

Reply:
No, it's writing that text to a .py file in an arbitrary directory, to test that we can import it from there.